<ui version="4.0" >
 <class>OmgModelTestsBase</class>
 <widget class="QDialog" name="OmgModelTestsBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>External Model Tests</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Tests properties</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" >
      <item row="0" column="0" >
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Label:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leTestName" />
        </item>
        <item>
         <widget class="QToolButton" name="toolModelSettings" >
          <property name="text" >
           <string>...</string>
          </property>
          <property name="icon" >
           <iconset resource="../resources/resources.qrc" >
            <normaloff>:/configure.png</normaloff>:/configure.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>leTestName</zorder>
     <zorder></zorder>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Occurrence data</string>
       </property>
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="0" >
         <widget class="OmgListWidget" native="1" name="omglistTaxa" />
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="MinimumExpanding" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Algorithm(s)</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2" >
        <item row="0" column="0" >
         <widget class="OmgListWidget" native="1" name="omglistAlgorithms" />
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <widget class="QProgressBar" name="progressBar" >
     <property name="value" >
      <number>0</number>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textVisible" >
      <bool>true</bool>
     </property>
     <property name="invertedAppearance" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbnRun" >
       <property name="text" >
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbnClose" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OmgListWidget</class>
   <extends>QWidget</extends>
   <header>omglistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc" />
 </resources>
 <connections/>
</ui>
